cloud_infrastructure_access/dedicated-understanding-aws.adoc

```diff
@@ -0,0 +1,25 @@
+:context: dedicated-understanding-aws
+[id="welcome-index"]
+= Understanding Cloud Infrastructure Access
+include::modules/common-attributes.adoc[]
+
+Amazon Web Services (AWS) infrastructure access allows
+link:https://access.redhat.com/node/3610411[Customer Portal Organization Administrators]
+and cluster owners to enable AWS Identity and Access Management (IAM) users to
+have federated access to the AWS Management Console for their {product-title}
+cluster.
+
+== Enabling AWS Access
+AWS access can be granted for customer AWS users, and private cluster access can be implemented to suit the needs of your {product-title} environment.
+
+Get started with xref:../cloud_infrastructure_access/dedicated-aws-access.adoc#dedicated-aws-access[Accessing AWS Infrastructure] for your {product-title} cluster. This will guide you through creating an AWS user and account and providing that user with access to the {product-title} AWS account.
+
+Once you have access to the {product-title} AWS account, there are multiple ways to establish private connections to your cluster:
+
+- xref:../cloud_infrastructure_access/dedicated-aws-peering.adoc#dedicated-aws-peering[Configuring AWS VPC Peering]: Enable VPC peering to route network traffic between two private IP addresses.
+
+- xref:../cloud_infrastructure_access/dedicated-aws-vpn.adoc#dedicated-aws-vpn[Configuring AWS VPN]: Establish a Virtual Private Network to securely connect your private network to your Amazon Virtual Private Cloud.
+
+- xref:../cloud_infrastructure_access/dedicated-aws-dc.adoc#dedicated-aws-dc[Configuring AWS Direct Connect]: AWS Direct Connect allows you to establish a dedicated network connection between your private network and an AWS Direct Connect location.
```

Comment:
s/AWS Direct Connect allows you/Configure AWS Direct Connect

> ! allow v
Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to...” Whenever
possible, use a direct, user-focused alternative such as “you can use the product to...” or “with this product,
users can...

There's also a good section in the ISG about avoiding anthropomorphism in general.